Lesson24/selcan1/index..html

```diff
@@ -0,0 +1,148 @@
+         }
+         @media all and (max-width: 576px) {
+        .kapsayici{
+            background-color: white;
+        }
+         .anaMenu{
+            flex-direction:column-reverse;
+            height:100%;
+            width: 100%;
+         }
+      }
+         .sagEkran{
+            display: flex;
+            margin-left: 90px;
+            justify-content: center;
+         }
+        .sagEkran>img{
+            width: 90%;
+            height: 100%;
+            border-radius:20px;
+        }
+         h2{
+            font-size: 30px;
+            padding: 0 20px;
+         }
+         ul{
+            line-height: 30px;
+            padding-left: 0;
+         }
+         .solEkran li{
+            list-style-type: none;
+            padding-left:10px;  
+         }
+         ul .kutu{
+            display: flex;
+            align-items: flex-start;
+         } 
+         .fa-circle-check{
+         color: red;
+         }
+         .form-input{
+            width: 100%;
+            height: 35px;
+            border-radius: 5px;
+            margin-bottom: 20px;
+         }
+         button{
+            width:calc(100% + 10px);
+            height: 50px;
+            background-color: rgb(4, 4, 30);
+            color: white;
+            padding: 5px auto;
+            border-radius: 10px;
+            cursor: pointer; 
+         }
+         button:hover{
+            background-color: #590808;
+            transition: .5s;
+         }
+        .error {
+            color: #590808;
+            font-size: 12px;
+            margin-top: 5px;
+            margin-bottom: 10px;
+            display: flex; 
+            justify-content: flex-end;  
+        }
+        .sayfa2{
+         display: none;
+        }
+       
+
+       
+    </style>
+</head>
+<body>
+    <div class="kapsayici">
+        
+       <div class="anaMenu">
+        <div class="solEkran">
+            <h2>Stay Updated!</h2>
+            <p>Join 60.0004 product managers receiving monthly updates on:</p>
+           <ul>
+               <div class="kutu"><i class="fa-solid fa-circle-check"></i><li>Product discovery and building what matters</li></div>  
+                <div class="kutu"><i class="fa-solid fa-circle-check"></i><li>Measuring to ensure updates are success </li></div>
+                <div class="kutu"><i class="fa-solid fa-circle-check"></i><li>And much more!</li></div>
+            </ul>
+            <form id="subscribeForm" novalidate>
+            <label for="email" class="form-label" >Email Address:</label>
+            <div id="emailError" class="error"></div>
+            <div id="emailError1" class="error1"></div>
+            <input type="email" id="email" name="email" class="form-input" placeholder="Email@company.com"required>
+            <div class="buton"><button>Subscribe to monthly newsletter</button></div></form>
+        </div>
+        <div class="sagEkran">
+            <img src="./1.png" alt="resim1">
```

Comment:
If it's hard for you to write a good alt text, use ChatGPT to generate it.
Don't user images in the .png format, it's very heavy. Convert the image to .jpg format and a size as big as you need for this page. It may be good to review resources in the course related to responsive images and responsive web-design in general.
```suggestion
            <img src="./1.png" alt="Illustration of overlapping digital interface elements on an orange background with graphs and a 94% progress bar.">
```